src/rt.rs

```diff
@@ -13,84 +13,90 @@
 // You should have received a copy of the GNU Lesser General Public License
 // along with rgtk.  If not, see <http://www.gnu.org/licenses/>.
 
-use libc::c_uint;
 use std::ptr;
 use ffi;
 use glib::translate::{FromGlibPtr};
 use glib::{to_bool, to_gboolean};
 
-pub fn init() {
-    unsafe {
-        ffi::gtk_init(ptr::null(), ptr::null());
+pub struct Gtk;
+
+impl Gtk {
+    pub fn new() -> Gtk {
```

Comment:
I guess better to return `Result<Gtk, GtkError>` or something here.
So when user try to call it the second time on a different thread, `unwrap` can print the error to show that it need a main thread to work.